modules/storages/app/components/storages/open_project_storage_modal_component.html.erb

```diff
@@ -3,8 +3,7 @@
                               title: 'Open project folder modal',
                               data:)
 ) do |dialog| %>
-    <% dialog.with_header(display: :none) %>
-    <% dialog.with_body(id: self.class.dialog_body_id, p: 2) do %>
```

Comment:
Hi @Kharonus , @akabiru 

Could one of you please have a look whether this still looks fine? I was not sure where this component is used 🙈

Reply:
Hi @HDinger , as discussed- it would seem the header display none was indeed relevant (oddly) - as there is now a programmatic header

<img width="1467" alt="Screenshot 2025-01-16 at 12 36 00 PM" src="https://github.com/user-attachments/assets/fe2a9451-11b3-4549-986d-ca7904282038" />